lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -888,10 +888,11 @@ void HdVP2Mesh::_UpdateDrawItem(
                 adjacency->GetSharedAdjacencyBuilderComputation(&topology);
             adjacencyComputation->Resolve(); // IS the adjacency updated now?
 
-            // The topology doesn't have to reference all of the points, thus
-            // we compute the number of normals as required by the topology.
+            // Only the points referenced by the topology are used to compute
+            // smooth normals.
             normals = Hd_SmoothNormals::ComputeSmoothNormals(
-                adjacency.get(), topology.GetNumPoints(),
+                adjacency.get(),
+                _meshSharedData._points.size(),
```

Comment:
The points array is empty because VP2RenderDelegate currently doesn't support USDSkel yet. If we passed in the number of points referenced by topology, it would end up with referencing a null array.

The original code tried to minimize the compute, but the same optimization has already been considered by the HdSmoothNormals::ComputeSmoothNormals() method at https://github.com/PixarAnimationStudios/USD/blob/master/pxr/imaging/hd/smoothNormals.cpp#L90, where 
`numAdjPoints` is the number of points referenced by topology and `numPoints` is the size of the points array. The minimum of the two will determine the size of the output normals.